File: scala-2.14/src/main/scala/cli/Main.scala
package cli

object Main {

  val runnies = scala.Vector(
    incubate3.Incubator,
    incubate4.Incubator
  )
  val pickRunny = 1
  def runny = runnies(pickRunny)

  def main(args: scala.Array[scala.Predef.String]): scala.Unit = {
    runny.run()
  }

}

File: scala-2.14/src/main/java/cli/Main.java
package cli;

import java.util.Vector;

public class Main {

    class Runnies {
        final Runnable runny0 = incubate4.Incubator;
    }

    Runnies allRunnies = new Runnies();
    Vector<Runnable> runnies = new Vector<>();

    public Main() {
        runnies.add(allRunnies.runny0);
    }

    public static void main(String[] args) {

    }

}
